### PR TITLE
Add example

### PR DESCRIPTION
**Pull request by ericphanson**
@SimonDanisch gave this example in the Slack for how to get the data coordinates for mouse interaction when using MakieLayout. I'm not quite sure how MakieGallery works so please tell me if I need to do something differently.

### CODE REVIEW

File: examples/interactive.jl [hunk 1]
```diff
@@ -56,6 +56,36 @@
         RecordEvents(final, @replace_with_a_path)
     end
 
+    @cell "Mouse hover with MakieLayout" [interactive] begin
+        import GLMakie
+        import GLMakie: AbstractPlotting
```
Comment (asinghvi17) [inline examples/interactive.jl:61]:
```suggestion
        using AbstractPlotting

File: examples/interactive.jl [hunk 2]
```diff
@@ -56,6 +56,36 @@
...
+        import .AbstractPlotting: lift, Point2f0
+        import .AbstractPlotting: Scene, mouseposition
```
Comment (asinghvi17) [inline examples/interactive.jl:63]:
```suggestion
```
All of these are exported already.

File: examples/interactive.jl [hunk 3]
```diff
@@ -56,6 +56,36 @@
...
+        layout[1,1] = laxis = LAxis(scene, title="workspace")
+        mouseposition(laxis.scene)
+        mousemarker = lift(laxis.scene.events.mouseposition) do mp
+            area = pixelarea(laxis.scene)[]
+            mp = Point2f0(mp) .- minimum(area)
+            r = [Point2f0(AbstractPlotting.to_world(laxis.scene, mp))]
+            return r
+        end
+        AbstractPlotting.scatter!(laxis, mousemarker; markersize=3)
+        display(scene)
```
Comment (asinghvi17) [inline examples/interactive.jl:82]:
```suggestion
        scene
```
Explicit displaying gets kinda annoying during CI ;)

**Comment by ericphanson** [comment]
Thanks for the review and fixes :)

**Comment by ericphanson** [comment]
One thing is that this doesn't use the `MakieLayout.addmousestate!` stuff which also gives a nice API for this. But this example might still be helpful anyway to show another way.

**Comment by asinghvi17** [comment]
Fair enough.  This is a pretty useful example IMO, just to show integration.  

I'll merge later this week when I have time to record this and upload the reference images.

**Pull request merged**